test/Scissorhands.Helpers.Tests/FileHelperTest.cs

```diff
@@ -47,7 +46,7 @@ public FileHelperTest(FileHelperFixture fixture)
         [Fact]
         public void Given_NullParameter_Constructor_ShouldThrow_ArgumentNullException()
         {
-            Action action = () => { var helper = new FileHelper(null); };
+            Action action = () => { var helper = new FileHelper(null, null); };
```

Comment:
1. There's no assert for the `action`.
1. As the constructor takes 2 parameters, we should check whether either parameter is null or not.